.github/workflows/triage.yml

```diff
@@ -1,127 +0,0 @@
-name: Add new issues to triage column
-
-on:
-  issues:
-    types:
-      - opened
-
-env:
-  GH_TOKEN: ${{ secrets.PROJECT_BOARD_AUTOMATION }}
```

Comment:
I guess we can now remove this secret? This seems to be the only workflow using it.

Reply:
yes :)

Reply:
Have you already enabled the automation in the workflow?

Reply:
yes I have. You can check it here: https://github.com/orgs/ethereum/projects/27/workflows